x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/asset_criticality/constants.ts

```diff
@@ -17,7 +17,7 @@ const assetCriticalityMapping = {
 };
 
 // Upgrade this value to force a mappings update on the next Kibana startup
-export const ASSET_CRITICALITY_MAPPINGS_VERSIONS = 2;
+export const ASSET_CRITICALITY_MAPPINGS_VERSIONS = 3;
```

Comment:
@abhishekbhatia1710 @machadoum - I told Abhishek to change this version because it was not working locally because the version on `main` was the same as on this branch.

Thinking about it I am not sure if this is correct, because the version hasnt been released yet (I dont think it has been released in serverless) maybe we should keep it as version 2?

Reply:
I agree with bumping it in this case so that the feature works consistently.